assets/js/just-the-docs.js

```diff
@@ -48,19 +48,19 @@ function initNav() {
 
 function initSearch() {
   var request = new XMLHttpRequest();
-  request.open('GET', '{{ "assets/js/search-data.json" | absolute_url }}', true);
+  request.open('GET', '/assets/js/search-data.json', true);
```

Comment:
This change is needed to avoid cross-site requests (which fail) between the testing site (like https://deploy-preview-678--docs-rapids-ai.netlify.app/) and the main site (https://docs.rapids.ai). Otherwise our search data fails to load on the site previews.